modules/vye/lib/dgib/configuration.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Vye
+  module DGIB
+    class Configuration < Common::Client::Configuration::REST
+      def connection
+        @conn ||= Faraday.new(base_path, headers: base_request_headers, request: request_options) do |faraday|
+          faraday.use :breakers
+          faraday.ssl[:ca_file] = 'modules/vye/spec/fixtures/ICA11-RCA2-combined-cert.pem'
```

Comment:
This will need to be Settings.dgi.vye....{combined cert path}

Reply:
we made this change